src/edu/wright/cs/raiderplanner/model/Task.java

```diff
@@ -72,6 +75,24 @@ public boolean isCheckedComplete() {
 		return canCheckComplete() && checkedComplete;
 	}
 
+	/**
+	 * Used for deadline notifications.
+	 *
+	 * @return.
+	 */
+	public boolean weekNotificationSent() {
```

Comment:
This should probably be called `oneWeekNotificationSent()`. However, you have also made the notification intervals configurable, so the method names should perhaps be more generic.